syntax = "proto3";

package gopython;

// export PATH="$PATH:$(go env GOPATH)/bin"
// protoc --go_out=. --go_opt=paths=source_relative --go-grpc_out=. --go-grpc_opt=paths=source_relative proto/gopython/gopython.proto
// python3 -m grpc_tools.protoc -I./proto/gopython --python_out=./python --grpc_python_out=./python --proto_path=./proto/gopython ./proto/gopython/gopython.proto

option go_package = "cs425mp3/proto/gopython";

service GoPython {
    rpc SetBatchSize(BatchRequest) returns (BatchResponse) {}
}

message BatchRequest {
    int32 batch_size = 1;
}

message BatchResponse {
    string status = 1;
}